linkerd/tests/conftest.py

```diff
@@ -12,7 +12,8 @@
 @pytest.fixture(scope='session')
 def dd_environment():
     with terraform_run(os.path.join(get_here(), 'terraform')) as outputs:
-        kubeconfig = outputs['kubeconfig']['value']
+        # TODO: Remove str() when we drop Python 2
+        kubeconfig = str(outputs['kubeconfig']['value'])
```

Comment:
Why is that needed ? It seems working for `istio` case https://github.com/DataDog/integrations-core/blob/6c9d3a68eebc3fcf5d53a7b5ae850ff5ad3c6aa5/istio/tests/conftest.py#L32

Reply:
Dunno, it fails here

Reply:
What was the error ?